v3/onesignal-metabox/onesignal-metabox.js

```diff
@@ -32,4 +32,25 @@ window.addEventListener("DOMContentLoaded", () => {
     setDisplay(customiseWrap, customisePost.checked);
     setDisabled(customiseWrapChild, !customisePost.checked);
   });
+
+  // Watch for the Publish button to be added to the DOM
+  const observer = new MutationObserver((mutations, obs) => {
+    const publishButton = document.querySelector('.editor-post-publish-button__button');
```

Comment:
The publish button is targeted from the document body (`.editor-post-publish-button__button`) but this might be too brittle since we don't control the class values on that button.

We can scrap the entire JS side changes if we'd rather the checkbox just be unchecked once the page is refreshed, which matches the current behavior in v2.